ractor_cluster/src/protocol/control.proto

```diff
@@ -76,6 +76,10 @@ message NodeSessions {
     repeated auth.NameMessage sessions = 1;
 }
 
+// All state for initial sync has been pushed
+message InitialSyncDone {
```

Comment:
nit: Maybe a better control message would be "Connected". And we could just send a uint32 instead of defining a new structure type. Though I don't really care about the structure type.

Reply:
Or maybe "Ready"? But I'm open to leaving this name here as is as well

Reply:
"Ready" sounds good. I'd like to keep the structure type, it's more uniform and leaves door open for additional payload we can send along ready signaling. A empty message should just take 1 or 2 bytes, no better or worse than uint32.